### DISCUSSION TITLE
新人画师又来力

### DISCUSSION THREAD

**Discussion by aoaococo** [share121/inter-knot#2101] [其他]
嗨害嗨又是我新人画师嗷嗷，觉得好康的可以关注一下谢谢(●￣(ｴ)￣●)https://space.bilibili.com/476029
![柯蕾妲](https://github.com/user-attachments/assets/fe9304d4-a670-4d1a-a4d9-5f8f484c9ac3)

**Comment by wugan-bot156** [comment]
# 去米游社啊，这里的人太少了

**Reply by aoaococo** [reply]
都发了都发了~那边推送比较差，所以到处发发~

**Reply by wugan-bot156** [reply]
# 阿哲，这里人更少，其实米游社里也没多少人，~~不如就在b站发~~，emmm 你加油
